context/app/static/js/components/files/GlobusLink/style.js

```diff
@@ -6,18 +6,9 @@ const StyledTypography = styled(Typography)`
   margin: 0px ${(props) => props.theme.spacing(1)}px ${(props) => props.theme.spacing(1)}px 0px;
 `;
 
-const CenteredDiv = styled.div`
-  display: flex;
-  justify-content: center;
-  margin-top: ${(props) => props.theme.spacing(2)}px;
-`;
-
-const MarginTopDiv = styled.div`
-  margin-top: ${(props) => props.theme.spacing(2)}px;
-`;
-
 const Flex = styled.div`
   display: flex;
+  margin-top: ${(props) => props.theme.spacing(1)}px;
```

Comment:
There should only be extra margin for the support link.

Reply:
I'll remove margin-top, add conditional `<br>` instead.

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br

> Note: Do not use <br> to create margins between paragraphs; wrap them in <p> elements and use the CSS margin property to control their size.

Reply:
Line breaks are specifically meant to be used inside of a `<p>` or similar.